build.gradle

```diff
@@ -124,13 +124,12 @@ jacocoTestReport {
   }
 }
 
-project.tasks['sonarqube'].dependsOn test, integration
+project.tasks['sonarqube'].dependsOn jacocoTestReport
```

Comment:
would be nice if this is a separate PR ? we like bonuses but easier for tracking later if required

Reply:
i'm not seeing any benefit in separating this. very tiny change and already proven in different pr. very simple boy scout activity here. separate PRs waste time and resources

Reply:
+ commit log is very clear. pr is very tiny (once again)

Reply:
Point is not how big or small it is those are two different changes. Although you can figure out from log but that doesn't mean two unrelated changes should go into one :) Approving for now but in future it would be helpful